Formula/grpc.rb

```diff
@@ -21,25 +21,44 @@ class Grpc < Formula
 
   depends_on "autoconf" => :build
   depends_on "automake" => :build
+  depends_on "cmake" => :build
   depends_on "libtool" => :build
+  depends_on "abseil"
   depends_on "c-ares"
   depends_on "gflags"
   depends_on "openssl@1.1"
   depends_on "protobuf"
 
-  resource "gtest" do
-    url "https://github.com/google/googletest/archive/release-1.10.0.tar.gz"
-    sha256 "9dc9157a9a1551ec7a7e43daea9a694a0bb5fb8bec81235d8a1e6ef64c716dcb"
-  end
-
   def install
-    system "make", "install", "prefix=#{prefix}"
+    mkdir "cmake/build" do
+      args = %w[
+        ../..
+        -DBUILD_SHARED_LIBS=ON
+        -DgRPC_BUILD_TESTS=OFF
+        -DgRPC_INSTALL=ON
+        -DgRPC_ABSL_PROVIDER=package
+        -DgRPC_CARES_PROVIDER=package
+        -DgRPC_PROTOBUF_PROVIDER=package
+        -DgRPC_SSL_PROVIDER=package
```

Comment:
Are these upstream's way of defining system dependencies?

Reply:
Yes. See https://github.com/grpc/grpc/blob/master/BUILDING.md#dependency-management

Reply:
Just checking since package could be explained either way without context.